build.sbt

```diff
@@ -117,6 +117,10 @@ lazy val spark = (project in file("spark"))
     ),
     // For adding staged Spark RC versions, Ex:
     // resolvers += "Apche Spark 3.5.0 (RC1) Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1444/",
+
+    // Add Iceberg 1.4.0 RC2 dep; TODO: replace with final release
+    resolvers += "Apache Iceberg 1.4.0 (RC2) Staging" at "https://repository.apache.org/content/repositories/orgapacheiceberg-1146/",
```

Comment:
Is there any risk or benefit to also adding
- https://repository.apache.org/content/repositories/orgapacheiceberg-1147
- https://repository.apache.org/content/repositories/orgapacheiceberg-1148
- https://repository.apache.org/content/repositories/orgapacheiceberg-1149

Reply:
do not think there is particular risk or benefit. will leave as it is. we should be able to use a non-staging iceberg very soon.